### ISSUE TITLE
Include Non-English Cards

### ISSUE THREAD

**Issue by JamesDonnelly** [PokemonTCG/pokemon-tcg-api#78]
This is a pretty large request, and I imagine not one which will happen any time soon: I'd love to be able to stick a flag on the API to return **ALL** language variants of each Pokémon card. 

---

Take Pichu, for example. In English, we have this POP Series 9 card:

![https://images.pokemontcg.io/pop9/11.png](https://images.pokemontcg.io/pop9/11.png)

The Japanese DP-P promo set features the same card, but holographic:

![https://cdn.bulbagarden.net/upload/0/0c/020004_P_PICHU.jpg](https://cdn.bulbagarden.net/upload/0/0c/020004_P_PICHU.jpg)

There is no holographic English version, and likewise there is no non-holographic Japanese version.

Interestingly, the French version of this card is also part of the POP Series 9 set with the same number, but theirs is the holographic version.

**Comment by adback03** [comment]
It's worth noting I have been working on foreign language support, and I expect to initially start with French as a beta, as I have many French users willing to test.

**Comment by joergl87** [comment]
I would love to see also german support.

Do you crawl the data from pokemon website? Seems the html format is identical?!

[https://www.pokemon.com/de/pokemon-sammelkartenspiel/pokemon-karten/platinum-series/pl1/130/](https://www.pokemon.com/de/pokemon-sammelkartenspiel/pokemon-karten/platinum-series/pl1/130/)

[https://www.pokemon.com/us/pokemon-tcg/pokemon-cards/platinum-series/pl1/130/](https://www.pokemon.com/us/pokemon-tcg/pokemon-cards/platinum-series/pl1/130/)

**Comment by lfgcampos** [comment]
There is another project where people get the same json structure for other languages.
ex: https://github.com/kirbyUK/ptcgo-data

**Comment by badetitou** [comment]
Hi. 

What is the status of the api to request french cards / or other languages?
I've seen that one json file already exists. I'd love to make pull request to add new cards. But I don't know of I'll be able to use the api to request the fr version?

**Comment by adback03** [comment]
@badetitou I'm hopeful for a v2 release still this year, and then the first couple major additions to the project after v2 comes out will be other languages and prices. The work to include foreign languages is paused for v1, as the json schema will be changing a bit for v2. I've recently come back from a much needed vacation from programming, and am about to get back into this project a lot over the next coming months.

**Comment by tyler-wel** [comment]
@adback03 
I'd be willing to help support Japanese language. Any ETA on v2? With the release of v2, if you accept outside help for translations, it would be helpful to have a tiny "contributing" write up on how to add json and image data.

On a side note, the Japanese card search site does actually have a hittable API that takes in form data, still playing with it
website: https://www.pokemon-card.com/card-search/
api: https://www.pokemon-card.com/card-search/resultAPI.php

**Comment by mbret** [comment]
I am also willing to help support japanese language. This is a mess since we do not use the same sets as you guys. For the same english set there are many sub smaller variant in japan for example.

I currently have an app we use with my friends to quickly get info about japanese cards and we use this API enriched with scrapping/crawling from bulbapedia for jp info. cf https://github.com/mbret/pokemax

I would happily contribute to this project and use the API directly rather than doing things on my side.

**Comment by badetitou** [comment]
Hi @adback03!

Congrats for the v2 that is out!
It is amazing!!!

I did not see in the documentation information about including foreign language. Is it still in the plan?

**Comment by adback03** [comment]
@badetitou yes, foreign language support is still on the radar. That will be one of the first enhancements to v2, now that it is released.

**Comment by zbayoff** [comment]
Any updates for this request?

**Comment by Askeo** [comment]
German and JPN would be so nice

**Comment by kirushi** [comment]
Please support japanese too!! <3

**Comment by FrankJeager** [comment]
I hope french support is always planned. I am very interested.

Thank you for your hard work.

**Comment by stepanzalis** [comment]
Any update on this?

**Comment by kaiserkiwi** [comment]
I'm interested in this too.

**Comment by DrSkyfaR** [comment]
I would Love to See this too.

**Comment by jeppelykkebjerg** [comment]
Japanese support would be amazing!

**Comment by knuxyl** [comment]
TCGdex has japanese but their images and api aren't as polished as this one. I'm hoping this is done soon. LimitlessTCG has a ton of newer japanese card images with translations.
